### PR TITLE
Child bounties comments

### PR DESCRIPTION
**Pull request by NukeManDan**
Only comment changes, no logic impacted.

* Use uniform notion of parent and child, no "master" or "general" entity referenced.
* README updated to match comments.

### CODE REVIEW

File: frame/child-bounties/src/lib.rs [hunk 1]
```diff
@@ -189,14 +189,14 @@ pub mod pallet {
...
-	/// Number of child-bounties per parent bounty.
+	/// Number of child bounties per parent bounty.
 	/// Map of parent bounty index to number of child bounties.
 	#[pallet::storage]
 	#[pallet::getter(fn parent_child_bounties)]
 	pub type ParentChildBounties<T: Config> =
 		StorageMap<_, Twox64Concat, BountyIndex, u32, ValueQuery>;
 
-	/// Child-bounties that have been added.
+	/// child bounties that have been added.
```
Comment (NukeManDan) [inline frame/child-bounties/src/lib.rs:199]:
```suggestion
	/// Child bounties that have been added.
```

File: frame/bounties/README.md [hunk 2]
```diff
@@ -2,43 +2,55 @@
...
+election. The Council also gets to cancel the bounty if deemed necessary before assigning a
+curator or once the bounty is active or payout is pending, resulting in the slash of the
+curator's deposit.
+
+This pallet may opt into using a [`ChildBountyManager`] that enables bounties to be split into
+sub-bounties, as children of anh established bounty (called the parent in the context of it's
+children).
+
+> NOTE: The parent bounty cannot be closed if it has a non-zero number of it has active child
+> bounties associated with it.
```
Comment (NukeManDan) [inline frame/bounties/README.md:26]: needs a check on my understanding

Comment (kianenigma) [reply frame/bounties/README.md:26]: You are right: https://github.com/paritytech/substrate/blob/26ea75756b44de8b83f927c63c6ed46a0f040386/frame/bounties/src/lib.rs#L668

**Review by kianenigma — APPROVED** [review]

**Review by bkchr — APPROVED** [review]

**Comment by bkchr** [comment]
bot merge

**Comment by bkchr** [comment]
bot merge

**Pull request merged**